apps/nuxt3-ssr/components/NetworkCard.vue

```diff
@@ -35,28 +33,38 @@ const iconStarClasses = computed(() => {
   return props.compact ? "" : "items-baseline xl:items-center mt-0.5 xl:mt-0";
 });
 
-const isShowingMobileMoreText = computed(() => {
-  return (
-    mobileShowMoreText.value ||
-    (props.network?.description &&
-      props.network?.description?.length < mobileShowMoreTextLength)
-  );
-});
+const links = [
+  {
+    title: "Network",
+    url: `/${props.schema}/ssr-catalogue/networks/${props.network.id}`,
+  },
+  {
+    title: "Cohorts",
```

Comment:
we also need databanks, datasources, etc. ideally we would find a term (in our data model is it 'data resources') that we could use to describe all of them. What I am trying to say: we need either more links or a more general listing.